### PR TITLE
Creating booklist array

### PR DESCRIPTION
**Pull request by MayPyone**
-Used webpack template.
-Used local storage and array to store objects.
-Added a function to iterate all objects.

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi  @MayPyone , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Your repo is missing a readme file. Kindly add it for documentation purposes

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/input.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  for (let i = 0; i < totaltasks.length; i += 1) {
+    inner.innerHTML += `
+       <div class="container"> 
+       <input type="checkbox" id="vehicle2" name="tasks" value="data">
+       <input class="data-input" value= "${totaltasks[i].des}">
+       <i class="fa-solid fa-ellipsis-vertical"></i>
+       </div>
+        `;
+  }
+};
```
Comment (AdnanOlarmmi) [inline src/input.js]: - [ ] According to the requirement, you should have hardcoded data in your code for now, to execute the other requirements, You probably do not need the localstorage at this point, all that is required is just the app structure, like

```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
]
```

Then task are supposed to be rendered dynamically on the DOM on load.

File: src/input.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  });
+};
+
+export const display = () => {
+  const inner = document.querySelector('.show');
+  const totaltasks = create();
+  for (let i = 0; i < totaltasks.length; i += 1) {
+    inner.innerHTML += `
+       <div class="container"> 
+       <input type="checkbox" id="vehicle2" name="tasks" value="data">
```
Comment (AdnanOlarmmi) [inline src/input.js]: - [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each object, meaning it should be checked for `completed: true` and unchecked for `completed: false`...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ' '}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox when it is present.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @MayPyone, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Working app.
- Stuck to styles.

## Required Changes ♻️
_Check the comments under the review._
- [ ] The `cd` command in the Readme should be directed to your repo name in this case `cd to-do-list`
image

## Optional suggestions
- [ ] I feel you could do a better job with regards naming your files 😃.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/input.js [hunk 3]
```diff
@@ -21,14 +21,28 @@ export const mydata = () => {
...
+    description: 'do homework',
+    completed: false,
+    index: 2,
+  },
+];
+
 export const display = () => {
   const inner = document.querySelector('.show');
-  const totaltasks = create();
+  // const totaltasks = create();
```
Comment (zainsadaqat) [inline src/input.js:39]: ### [OPTIONAL]
Please remove the commented code to make your codebase looks clean and professional 👍

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @MayPyone  👋 

### Things to appreciate
- [ ] Good job for implementing the things requested by the previous code reviewers. ✅ 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**